core/src/main/resources/jenkins/model/GlobalCloudConfiguration/index.groovy

```diff
@@ -21,25 +22,31 @@ l.layout(norefresh:true, permission:app.ADMINISTER, title:my.displayName) {
             // TODO more appropriate icon
             text(my.displayName)
         }
-
         def clouds = Cloud.all()
-
         if (!clouds.isEmpty()) {
             p()
             div(class:"behavior-loading", _("LOADING"))
 
             f.form(method:"post",name:"config",action:"configure") {
                 f.block {
+                    if (app.clouds.size() == 0 && !h.hasPermission(app.ADMINISTER)) {
+                        p(_("No clouds have been configured."))
+                    }
+
                     f.hetero_list(name:"cloud", hasHeader:true, descriptors:Cloud.all(), items:app.clouds,
                             addCaption:_("Add a new cloud"), deleteCaption:_("Delete cloud"))
                 }
 
-                f.bottomButtonBar {
-                    f.submit(value:_("Save"))
-                    f.apply(value:_("Apply"))
+                l.isAdmin {
+                    f.bottomButtonBar {
+                        f.submit(value: _("Save"))
+                        f.apply(value: _("Apply"))
+                    }
                 }
             }
-            st.adjunct(includes: "lib.form.confirm")
+            l.isAdmin {
+                st.adjunct(includes: "lib.form.confirm")
+            }
         } else {
             String label = Jenkins.get().updateCenter.getCategoryDisplayName("cloud")
 
```

Comment:
The link to the plugin manager two lines below should probably be conditional on having Administer? While the link isn't broken, it's not _that_ useful. OTOH it would inform which plugins need installing out of band. Unsure.

Reply:
I think it's fine as it allows the user to find the plugins they could get installed